src/pitchinterface.cpp

```diff
@@ -70,4 +70,25 @@ void PitchInterface::AdjustPname(int &pname, int &oct)
     }
 }
 
+void PitchInterface::AdjustPitchByOffset(int pitchOffset)
+{
+    int pname = this->GetPname() + pitchOffset;
+    int oct = this->GetOct();
+    
+    // Check if a change in octave is necessary
+    while (pname > PITCHNAME_b)
```

Comment:
You probably want to check the octave range (0..9) see http://music-encoding.org/guidelines/dev/data-types/data.octave.html

Reply:
Oh yeah I'll check that.